pkg/cli/admin/release/mirror.go

```diff
@@ -890,14 +897,41 @@ func printImageContentInstructions(out io.Writer, from string, toList []string,
 	sources = uniqueSources
 
 	// Create and display install-config.yaml example
+	v, err := semver.Parse(version)
```

Comment:
Is this server version or client?

Reply:
The command does not hit the server. It's openshift release version of the passed-in argument release image.